test/functional/page_objects/time_picker.ts

```diff
@@ -116,23 +116,38 @@ export class TimePickerPageObject extends FtrService {
   public async setAbsoluteRange(fromTime: string, toTime: string) {
     this.log.debug(`Setting absolute range to ${fromTime} to ${toTime}`);
     await this.showStartEndTimes();
+    let panel!: WebElementWrapper;
```

Comment:
Just a nit:
Do we need the non-null assertion operator here?
I don't think WebElementWrapper can be null.
Did your compiler complain?

Reply:
Type checker on CI is not happy without it :)